src/components/dev-hub/select.js

```diff
@@ -78,6 +78,7 @@ const FormSelect = ({
     errors = [],
     narrow = false,
     onChange = null,
+    styleSelectedText = null,
```

Comment:
This prop helps add additional formatting to the selected text, so we can provide it different styling (such as removing the number) if need be. This addresses the hiding numbers issue in the dropdown.